moveit_ros/planning/kinematics_plugin_loader/src/kinematics_plugin_loader.cpp

```diff
@@ -344,14 +344,6 @@ robot_model::SolverAllocatorFn KinematicsPluginLoader::getLoaderFunction(const s
             }
           }
 
-          std::string ksolver_attempts_param_name;
-          if (nh.searchParam(base_param_name + "/kinematics_solver_attempts", ksolver_attempts_param_name))
```

Comment:
This seems like it could be left in with an warning message that setting this parameter is no longer supported. Please include a comment with a link to this PR and a date so that future us know when we are able to delete it.

Reply:
Good idea.

Reply:
Addressed.